doctor_visits/delphi_doctor_visits/patch.py

```diff
@@ -0,0 +1,48 @@
+"""
+This module is used for patching data in the delphi_doctor_visits package.
+
+To use this module, you need to specify the range of issue dates in params.json.
+
+It will generate data for that range of issue dates, and store them in batch issue format:
+[name-of-patch]/issue_[issue-date]/doctor-visits/actual_data_file.csv
+"""
+
+from datetime import datetime, timedelta
+from os import makedirs
+
+from delphi_utils import get_structured_logger, read_params
+
+from .run import run_module
+
+if __name__ == "__main__":
+    # Run the doctor visits indicator for a range of issue dates, specified in params.json using following keys:
+    # - "patch": Only used for patching data
+    #     - "start_date": str, YYYY-MM-DD format, first issue date
+    #     - "end_date": str, YYYY-MM-DD format, last issue date
+    #     - "patch_dir": str, directory to write all issues output
+    params = read_params()
+    logger = get_structured_logger(__name__, filename=params["common"]["log_filename"])
```

Comment:
It seems that the first few lines when the code is running directly from patch.py the logger is set to "__main__"  but when it starts to go to run_module (it reverts back into delphi_doctor_visits.run) 

I'm not sure how to approach this such that the logger value stays consistent. Haven't done any fancy logging stuff myself yet. @melange396 any ideas?

Reply:
yup, cuz [`run_module()` creates its own logger that it uses](https://github.com/cmu-delphi/covidcast-indicators/blob/5c3521186401243c737bb6465cffcd4f1d3e9c11/doctor_visits/delphi_doctor_visits/run.py#L52-L54)

Reply:
logger fixed: [f68009a](https://github.com/cmu-delphi/covidcast-indicators/pull/1977/commits/f68009a5f00015046da6239566bdb81f970f572d)